app/src/main/java/com/portfolio/tasky/entry/usecases/PasswordPatternValidation.kt

```diff
@@ -1,8 +1,8 @@
-package com.portfolio.tasky.usecases
+package com.portfolio.tasky.entry.usecases
 
 import javax.inject.Inject
 
-class PasswordPatternValidation @Inject constructor(){
+class PasswordPatternValidation @Inject constructor() {
     fun isPasswordPatternValid(password: CharSequence): Boolean {
         return Regex("^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{9,}\$").matches(password)
     }
```

Comment:
I would personally avoid regex if not really necessary. Nobody will be able to quickly tell you what this matches for. A function like this would be much more readable:
```
val hasValidLength = password.length in (MIN_CHARACTERS_FOR_PASSWORD..MAX_CHARACTERS_FOR_PASSWORD)
val hasUppercase = password.any { it.isUppercase() }
val hasLowercase = password.any { it.isLowercase() }
val hasDigits = password.any { it.isDigit() }

return hasValidLength && hasUppercase && hasLowercase && hasDigits
```

Reply:
True

Thanks for providing a better readable fun! Done